cMake/FindOCC.cmake

```
@@ -7,164 +7,134 @@
 
 # handle the QUIETLY and REQUIRED arguments and set OCC_FOUND to TRUE if
 # all listed variables are TRUE
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(OCC REQUIRED_VARS OCC_INCLUDE_DIR VERSION_VAR OCC_VERSION_STRING)
+find_package_handle_standard_args(OCC REQUIRED_VARS OCC_INCLUDE_DIR VERSION_VAR OCC_VERSION_STRING)
 
-if(OCC_FOUND)
-  set(OCC_LIBRARIES
-    TKFillet
-    TKMesh
-    TKernel
-    TKG2d
-    TKG3d
-    TKMath
-    TKShHealing
-    TKXSBase
-    TKBool
-    TKBO
-    TKBRep
-    TKTopAlgo
-    TKGeomAlgo
-    TKGeomBase
-    TKOffset
-    TKPrim
-    TKHLR
-    TKFeat
-  )
-  set(OCC_OCAF_LIBRARIES
-    TKBin
-    TKBinL
-    TKCAF
-    TKXCAF
-    TKLCAF
-    TKVCAF
-    TKCDF
-    TKMeshVS
-    TKService
-    TKV3d
-  )
+if (OCC_FOUND)
+    set(OCC_LIBRARIES
+            TKFillet
+            TKMesh
+            TKernel
+            TKG2d
+            TKG3d
+            TKMath
+            TKShHealing
+            TKXSBase
+            TKBool
+            TKBO
+            TKBRep
+            TKTopAlgo
+            TKGeomAlgo
+            TKGeomBase
+            TKOffset
+            TKPrim
+            TKHLR
+            TKFeat
+    )
+    set(OCC_OCAF_LIBRARIES
+            TKBin
+            TKBinL
+            TKCAF
+            TKXCAF
+            TKLCAF
+            TKVCAF
+            TKCDF
+            TKMeshVS
+            TKService
+            TKV3d
+    )
 
-  if(NOT OCC_VERSION_STRING VERSION_LESS 7.5.0)
-    list(APPEND OCC_OCAF_LIBRARIES TKRWMesh)
-  endif(NOT OCC_VERSION_STRING VERSION_LESS 7.5.0)
-  if(OCC_VERSION_STRING VERSION_LESS 7.8.0)
-    list(APPEND OCC_LIBRARIES TKIGES TKSTL TKSTEPBase TKSTEPAttr TKSTEP209 TKSTEP)
-    list(APPEND OCC_OCAF_LIBRARIES TKXDESTEP TKXDEIGES)
-  else(OCC_VERSION_STRING VERSION_LESS 7.8.0)
-    list(APPEND OCC_LIBRARIES TKDESTEP TKDEIGES TKDEGLTF TKDESTL)
-  endif(OCC_VERSION_STRING VERSION_LESS 7.8.0)
-  message(STATUS "-- Found OCE/OpenCASCADE version: ${OCC_VERSION_STRING}")
-  message(STATUS "-- OCE/OpenCASCADE include directory: ${OCC_INCLUDE_DIR}")
-  message(STATUS "-- OCE/OpenCASCADE shared libraries directory: ${OCC_LIBRARY_DIR}")
-else(OCC_FOUND)
-  #message(SEND_ERROR "Neither OpenCASCADE Community Edition nor OpenCasCade were found: will not build CAD modules!")
-endif(OCC_FOUND)
+    if (NOT OCC_VERSION_STRING VERSION_LESS 7.5.0)
+        list(APPEND OCC_OCAF_LIBRARIES TKRWMesh)
+    endif ()
+    if (OCC_VERSION_STRING VERSION_LESS 7.8.0)
+        list(APPEND OCC_LIBRARIES TKIGES TKSTL TKSTEPBase TKSTEPAttr TKSTEP209 TKSTEP)
+        list(APPEND OCC_OCAF_LIBRARIES TKXDESTEP TKXDEIGES)
+    else ()
+        list(APPEND OCC_LIBRARIES TKDESTEP TKDEIGES TKDEGLTF TKDESTL)
+    endif ()
+    message(STATUS "-- Found OCE/OpenCASCADE version: ${OCC_VERSION_STRING}")
+    message(STATUS "-- OCE/OpenCASCADE include directory: ${OCC_INCLUDE_DIR}")
+    message(STATUS "-- OCE/OpenCASCADE shared libraries directory: ${OCC_LIBRARY_DIR}")
```

Probably drop the reference to OCE here too:
```suggestion
    message(STATUS "-- Found OpenCASCADE version: ${OCC_VERSION_STRING}")
    message(STATUS "-- OpenCASCADE include directory: ${OCC_INCLUDE_DIR}")
    message(STATUS "-- OpenCASCADE shared libraries directory: ${OCC_LIBRARY_DIR}")
```